projects/plugins/jetpack/modules/search/instant-search/components/search-filters.scss

```diff
@@ -35,15 +35,15 @@
 
 	> div {
 		display: flex;
-		align-items: center;
+		align-items: baseline;
 		margin-top: 8px;
 	}
 
 	.jetpack-instant-search__search-filter-list-input,
 	// Extra specific to override in some themes
 	.widget_search .jetpack-instant-search__search-filter-list-input {
 		cursor: pointer;
-		top: 0; // Ensures align-items: center; works as expected.
+		top: 1px; // Ensures align-items: center; works as expected.
```

Comment:
Shall we remove the comment on this line? Seems outdated now we're using `align-items: baseline`.

Reply:
Yes, definitely - good catch @bluefuton . I've leant towards amending over removing, as it's still helpful to explain why this is in there. Do you think `// Ensures align-items: baseline; works as expected.` is descriptive enough/still makes sense?

Reply:
Sounds good to me! 👍